src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -16,14 +17,13 @@
                                 <a href="{{ profile.url }}">{{ profile.url }}</a>
                             {% else %}
                                 {{ profile.url }}
-                                {% set referer = profile.collector('request').requestheaders.get('referer') %}
+                                {% set referer = request_collector ? request_collector.requestheaders.get('referer') : null %}
                                 {% if referer %}
-                                    <a href="{{ referer }}" class="referer">Replay referer URL</a>
+                                    <a href="{{ referer }}" class="referer">Back to referer URL</a>
```

Comment:
May I suggest `Go to the referer URL`?

"back" suggest we'll use the back button of the browser (`window.history.back()`) which is not similar because using "back" doesn't reset the state of the page (for instance forms will still be filled, and it's not the case here).

Reply:
then i like to sugges `Return to referer URL` instead

Reply:
Ok for me. But what if the profiler is used in a non-webpage context? For instance to profile an API response?

Reply:
Yes, could still be useful to copy the link and paste it in Postman or a `curl` command.

Reply:
Updated.

Status: needs review